recipe/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "forex-python" %}
+{% set version = "1.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/MicroPyramid/forex-python/archive/v{{ version }}.tar.gz
+  sha256: 062adc6380b9ac49168a2b89faba2fa60fa5806175da61230b121f139c9c4ab5
+
+build:
+  script: {{ PYTHON }} -m pip install --no-deps --no-build-isolation . -vv
+  number: 0
+
+requirements:
+  host:
+    - python
+    - pip
+    - wheel
+    - setuptools
+
+  run:
+    - python
+    - requests
+    - simplejson
+
+test:
+  source_files:
+    - tests
+  requires:
+    - pip
+    - pytest
+    - requests
+  imports:
+    - forex_python
+  commands:
+    - pip check
+    #bitcoin test failing, see https://github.com/MicroPyramid/forex-python/issues/147#issuecomment-1782761297
+    #test.py failing https://github.com/MicroPyramid/forex-python/issues/148
+    # - pytest
```

Comment:
if tests are failing for specific reason, I think it's good to document them here and still run `pytest tests`.
Also, tests have been added as `source_files`.

Reply:
I will put it on hold until snowflake tell us what they want